external/concealer/include/concealer/service.hpp

```diff
@@ -30,113 +31,70 @@ namespace concealer
 template <typename T>
 class Service
 {
-  const std::string service_name;
-
-  rclcpp::Logger logger;
+  const std::string name;
 
   typename rclcpp::Client<T>::SharedPtr client;
 
-  rclcpp::WallRate validation_rate;
+  rclcpp::WallRate rate;
 
 public:
   template <typename Node>
   explicit Service(
-    const std::string & service_name, Node & node,
-    const std::chrono::nanoseconds validation_interval = std::chrono::seconds(1))
-  : service_name(service_name),
-    logger(node.get_logger()),
-    client(node.template create_client<T>(service_name, rmw_qos_profile_default)),
-    validation_rate(validation_interval)
+    const std::string & name, Node & node,
+    const std::chrono::nanoseconds & rate = std::chrono::seconds(1))
```

Comment:
"rate" is a keyword that generally has the dimension of the inverse of time, so do not confuse it with anything that has the dimension of time.

Reply:
[85a38ee](https://github.com/tier4/scenario_simulator_v2/pull/1547/commits/85a38eefbcbcc8c79ee1359c912d0b04f8282143) fixed this.

Reply:
Oh, A WallRate named "interval" was born...
But it is acceptable because this is not critical for users.